assemblies/assembly_openshift-authoring-managed/main.adoc

```diff
@@ -23,6 +23,17 @@ and other business assets.
 
 * {OPENSHIFT} version 3.11 is deployed.
 * At least four gigabytes of memory are available in the OpenShift cluster/namespace.
+* For a high-availability deployment, the following resources are available on the OpenShift cluster:
+** For the {CENTRAL} replicated pod, 8 gigabytes of memory and 2 CPU cores are required per replica and two replicas are created by default.
+** For the {KIE_SERVER} replicated pod, 1 gigabyte of memory and 1 CPU core are required per replica and two replicas are created by default.
+** For the Red Hat Data Grid replicated pod, 2 gigabytes of memory and 1 CPU core are required per replica and two replicas are created by default.
+** The Red Hat AMQ replicated pod uses the default resource limits configured on your cluster.
+
+[NOTE]
+====
+For instructions about checking the capacity of your cluster, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/cluster_administration/admin-guide-cluster-capacity[Analyzing cluster capacity] in the {OPENSHIFT} 3.11 product documentation.
```

Comment:
General it is better to link to the top level of an external document and then tell the user what section to go to. The author can move the section and break the link. Up to you.